plugins/plugin-chart-echarts/src/Timeseries/transformProps.ts

```diff
@@ -109,26 +112,41 @@ export default function transformProps(chartProps: ChartProps): EchartsTimeserie
             : 0,
       });
   });
+
+  // yAxisBounds sometimes starts returning NaNs, which messes up the u-axis
+  let [min, max] = (yAxisBounds || []).map((val?: number) =>
+    val !== undefined && Number.isNaN(val) ? undefined : val,
```

Comment:
Apparently TS doesn't allow for `Number.isNaN(undefined)`, hence the seemingly unnecessary check for `undefined` here..

Reply:
You can probably just do `yAxisBounds.map(Number)` and use `Number.isNaN` check later.

Reply:
Thanks, looks much better!